tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -108,6 +109,14 @@ private static unsafe void Main(string[] args)
         {
             PrintLine("SwitchOpDefault test: Ok.");
         }
+
+        var cpObjTestA = new TestValue { Field = 1234 };
+        var cpObjTestB = new TestValue { Field = 5678 };
+        CpObjTest.CpObj(ref cpObjTestB, ref cpObjTestA);
+        if (cpObjTestB.Field == cpObjTestA.Field)
```

Comment:
Please add a check that they're equal to the expected value (in other words, confirming that the copy went the right direction).

Reply:
Thanks for pointing that out, I've made the change.